example.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * This is a hypothetical implementation to understand
+ * what implementations may look like as we define
+ * a standard for video quality of experience metrics.
+ */
+
+// States
+var NOT_PLAYING = 0;
+var ATTEMPTING_TO_PLAY = 1;
+var PLAYING = 2;
+
+
+// Watch Time
+var watchTime = 0;
+var lastWatchTimeCheck;
+var watchTimeInterval;
+
+function updateWatchTime() {
+  watchTime = watchTime + (Date.now() - lastWatchTimeCheck);
```

Comment:
Do we want to be as accurate as possible here? It's probably not worth it, but we should probably do `var now = Date.now()` at the top and use that, or someone might claim we're off by a millisecond or two